### ISSUE TITLE
Support for schema enum extensions?

### ISSUE THREAD

**Issue by levibuzolic** [facebook/relay#3748]
https://github.com/facebook/relay/blob/14550c8bd80e53369852db66d7681f13d6dd5f84/compiler/crates/schema/src/in_memory/mod.rs#L732-L734

Is there already work in progress for the new Rust compiler to support `EnumTypeExtension` and some of the other features here marked as `todo`? We're looking to upgrade to Relay 13, but trying to work out if we'll need to work around the missing extension support here.

**Comment by alunyov** [comment]
Yeah, I think we should to add this. We don't use this internally, but we need this to complete the `schema` API.
